frontend/src/components/dashboard/geo_location/ButtonComponent.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <VBtn
+        :ripple="false"
+        text
+        rounded
+        :class="isActive ? 'cyan white--text active' : 'inactive'"
+        @click="changeParameter"
+    >
+        {{ label }}
+    </VBtn>
+</template>
+
+<script>
+    import {mapGetters} from 'vuex';
+    import {GET_SELECTED_PARAMETER} from "@/store/modules/geo_location/types/getters";
+
+    export default {
+        name: 'ButtonComponent',
+        props: {
+            label: {
+                type:String,
+                required: true
+            },
+            type: {
+                type:String,
+                required: true
+            },
+        },
+        computed: {
+            ...mapGetters('geo_location', {
+                getSelectedParameter: GET_SELECTED_PARAMETER,
```

Comment:
Probably it's better to be consistent here - or use vuex for getting active param and action/mutation to update it in this component, or use props and events only, no need to mix both approaches